src/app/collection/pages/collection-ncyte/feature/collection-learning-object-card/collection-learning-object-card.component.scss

```diff
@@ -0,0 +1,65 @@
+@import '~_vars.scss';
+.container{
+    width: 20vw;
+    display: flex;
+    margin-bottom: 50px;
+}
+
+.module-background{
+    position: relative;
+    margin: auto;
+    right: 200px;
+}
+
+.module-background .background{
+    min-width: 100%;
+    min-height: 100%;
+}
+
+.module{
+    position: absolute;
+    bottom: 40px;
+    left: 50px;
+    z-index: 1;
+    color: #fff;
+    width: 100%;
+
+}
+.micromodule{
+    position: absolute;
+    left: 40px;
+    bottom: 40px;
+    z-index: 1;
+    color: #fff;
+}
+div{ 
+    font-size: $smaller;
+    padding-right: 50px;
+}
+
+.obj-info{
+    position: relative;
+    right: 250px;
+    margin: auto;
+    min-height: 10vh;
+
+}
+
+.obj-title{
+    font-weight: 700;
+}
+
```

Comment:
I would recommend adding 
.desc {
     margin: 0 0 0 15px;
}
@media and screen(width: ___) { //I don't know what width it would be good to do this at
     margin: 15px 0 0 0 !important;
}